packages/sui-mono/README.md

```diff
@@ -205,7 +205,7 @@ By default packages will be published as `restricted` in npm. If you want them t
 
 ### Scope (`packagesFolder`, `deepLevel`, `customScopes`)
 
-> 👉 `` Setting the proper scope in the commit message is important, because this is used for `sui-mono check` and `sui-mono release` to assign changes to specific packages and release them to the proper packages
+> 👉 ``Setting the proper scope in the commit message is important, because this is used for`sui-mono check`and`sui-mono release` to assign changes to specific packages and release them to the proper packages
```

Comment:
you sure you want to remove the whitespaces, ok for the first one but why removing the rest? 🤔

Reply:
I dnt know why are we starting with '``' symbols, but the spaces around the and are well done

Reply:
You're right! Maybe it's better to remove the first quotation marks (``) and still having white spaces around `and` word. I think there was a mess on linting with the opened first quotation marks. 🤔

```suggestion
> 👉 Setting the proper scope in the commit message is important, because this is used for `sui-mono check` and `sui-mono release` to assign changes to specific packages and release them to the proper packages
```